pootle/core/initdb.py

```diff
@@ -315,3 +321,42 @@ def create_default_admin():
     admin = User(**criteria)
     admin.set_password("admin")
     admin.save()
+
+
+def create_quality_checks_descriptions():
+    """Create the descriptions for all the quality checks."""
+    # First get Evernote quality checks descriptions.
+    filename = os.path.join(os.path.abspath(os.path.dirname(__file__)),
+                            'quality_checks_descriptions.html')
+
+    with codecs.open(filename, 'r', 'utf-8') as f:
+        body = f.read()
+
+    # Now get Translate Toolkit quality checks descriptions.
+
+    def get_check_description(name, filterfunc):
+        # Provide a header with an anchor to refer to.
+        description = ('\n<h3 id="%s">%s</h3>\n\n' %
+                       (name, unicode(check_names[name])))
+
+        # Clean the leading whitespace on each docstring line so it gets
+        # properly rendered.
+        docstring = filterfunc.__doc__.replace('        ', '')
```

Comment:
Will use `'\n'.join([line.strip() for line in filterfunc.__doc__.split('\n')])` instead in order to get exactly the same result as previously. This keeps the paragraphs, resulting in a more readable description.